packages/core/src/interpreter.ts

```diff
@@ -200,7 +200,9 @@ export class Interpreter<
       warn(
         this.initialized,
         // tslint:disable-next-line:max-line-length
-        `Attempted to read initial state from uninitialized service '${this.id}'. Make sure the service is started first.`
+        `Attempted to read initial state from uninitialized service '${
+          this.id
+        }'. Make sure the service is started first.`
```

Comment:
I wonder why Prettier keeps going back and forth between this

Reply:
not sure, was wondering about the same thing - maybe some version mismatch? but precommit hook should use a stable version, so it's rather unlikely

if that is going to happen more often then I'm going to investigate this as it's super noisy and distracting